RELEASING.md

```diff
@@ -45,6 +55,26 @@ mvn clean release:prepare -DautoVersionSubmodules=true -Prelease
 ```
 
 
+To upload it, perform this command:
+
+```sh
+mvn release:perform -Prelease
+```
+
+
+### Resuming release upload
+
+If something happened during the release upload you may need to eventually redo the upload.
+
+There is a release.properties file that is generated during the release. In case you want to upload a previously tagged release, add this file as follows:
+
+- release.properties
+```
+scm.url=scm:git:https://github.com/apache/activemq-artemis.git
```

Comment:
Ah.  I misread what you meant by release upload... you're talking about the upload to nexus.